config/components/webhook/manifests.yaml

```diff
@@ -365,8 +365,10 @@ webhooks:
     - v1beta1
     operations:
     - CREATE
+    - UPDATE
     resources:
     - workloads
+    - workloads/status
```

Comment:
I see what you do so - to default the LastTransitionTime. enabling webhooks for this reason for updates feels like overkill. So, I'm rather leaning to keep it as is wrt to LastTransitionTime.

In the future we will have MAP (mutating admission policies). This will be a better task for them.

Reply:
I see and it makes sense to me 👍 
I would keep the API change in the PR for now and move the other mutations to the reconciler.

Reply:
Yes, moving this code to reconcilers and minimizing the diff would help a lot to review. I will give another pass when this is addressed.